Changes.md

```diff
@@ -10,6 +10,7 @@
 - Technical
    - Run internal `make` and $(MAKE) to ensure that parameters are propagated correctly ([issue](https://github.com/INCATools/ontology-development-kit/issues/554)).
    - Mirror goal is only run when mirrors have been downloaded ([pull](https://github.com/INCATools/ontology-development-kit/pull/570))
+   - src/ontology/run.sh can now be configured with a MEMORY_JAVA parameter. It also warns the user in certain cases to update their local ODK image.
```

Comment:
This does not accurately reflect the way the configuration works in the `run.sh` script.

The max memory can be configured with a `robot_java_args` parameter in the ODK configuration file, _or_ with a `MEMORY_GB` (which should be renamed to `ODK_MEMORY_GB`, see other comment) environment variable.

Reply:
Updated text